astropy/io/votable/tree.py

```diff
@@ -4174,6 +4174,8 @@ def config(self):
         """
         Configuration used to construct this object. Will always include the
         version check values.
+
+        .. versionadded:: 7.0.0
```

Comment:
this is from https://github.com/astropy/astropy/pull/18366
```suggestion
        .. versionadded:: 7.2.0
```